GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -572,5 +615,77 @@ private void saveAsToolStripMenuItem_Click(object sender, EventArgs e)
                 }
             }
         }
+
+        private string GetSelectedFileInFileTree()
```

Comment:
I'd shorten the method name to ` GetSelectedFile` since it is being executed in the context  of file tree.

Reply:
Indeed, now that the Revsion tree is in its own file.
The development was made before that ;)
I will rename it.